pkgs/applications/networking/instant-messengers/rambox-pro/default.nix

```diff
@@ -0,0 +1,46 @@
+{ autoPatchelfHook, electron, fetchurl, makeDesktopItem, makeWrapper, nodePackages, nss, stdenv, xdg_utils, xorg }:
+
+stdenv.mkDerivation rec {
+  name = "rambox-pro-${version}";
```

Comment:
Could use `pname = "rambox-pro"` here instead

Reply:
Ok, I've addressed this one. Thanks! :)